policyengine_us/variables/gov/states/mt/tax/income/mt_taxable_social_security.py

```diff
@@ -0,0 +1,79 @@
+from policyengine_us.model_api import *
+
+
+class mt_taxable_social_security(Variable):
+    value_type = float
+    entity = Person
+    label = "Montana taxable social security benefits"
+    defined_for = StateCode.MT
+    definition_period = YEAR
+    reference = "https://mtrevenue.gov/wp-content/uploads/mdocs/form%202%202021.pdf#page=6"
+
+    def formula(person, period, parameters):
+        p_amount = parameters(
+            period
+        ).gov.states.mt.tax.income.social_security.amount
+        p_rate = parameters(period).gov.irs.social_security.taxability.rate
```

Comment:
```suggestion
        p_irs = parameters(period).gov.irs.social_security.taxability.rate
```

Reply:
Lets use "p" for the state parameter path and "p_irs" for the federal parameter path